Packs/MISP/Integrations/MISPV3/MISPV3_test.py

```diff
@@ -719,3 +719,51 @@ def test_add_msg_email_object(mocker):
     with pytest.raises(DemistoException) as exception_info:
         add_email_object(demisto_args)
     assert 'misp-add-email-object command does not support *.msg files' in str(exception_info.value)
+
+
+@pytest.mark.parametrize(
+    'demisto_args, is_attribute, expected_result',
+    [
+        (
+            {'uuid': 'test_uuid', 'tag': 'test_tag', 'disable_output': True},
+            True,
+            {
+                'readable_output': 'Tag test_tag has been successfully added to attribute test_uuid',
+                'outputs': None,
+                'outputs_prefix': None,
+            }
+        ),
+        (
+            {'uuid': 'test_uuid', 'tag': 'test_tag', 'disable_output': False},
+            True,
+            {
+                'readable_output': 'Tag test_tag has been successfully added to attribute test_uuid',
+                'outputs': {'test': 'test'},
+                'outputs_prefix': 'MISP.Attribute',
+            }
+        ),
+        (
+            {'uuid': 'test_uuid', 'tag': 'test_tag', 'disable_output': False},
+            False,
+            {
+                'readable_output': 'Tag test_tag has been successfully added to event test_uuid',
+                'outputs': {'test': 'test'},
+                'outputs_prefix': 'MISP.Event',
+            }
+        )
+    ]
+)
+def test_add_tag(demisto_args: dict, is_attribute: bool, expected_result: dict, mocker):
+
+    mock_misp(mocker)
+    from MISPV3 import add_tag
+    from pymisp import ExpandedPyMISP
+    mocker.patch.object(ExpandedPyMISP, 'tag', return_value={})
+    mocker.patch.object(ExpandedPyMISP, 'search', return_value={})
+    mocker.patch('MISPV3.build_attributes_search_response', return_value={'test': 'test'})
+    mocker.patch('MISPV3.build_events_search_response', return_value={'test': 'test'})
+    result = add_tag(demisto_args, is_attribute=is_attribute)
+
```

Comment:
please assert that `PYMISP.search` is called if and only if necessary. 
You can add a `mocked_search =` prefix to line where it's mocked, then `assert bool(mocked_search.call_count)` (or something like that) `== not disable_output`